src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -227,6 +227,9 @@ internal sealed class EventSourceAutoGenerateAttribute : Attribute
     /// }
     /// </code>
     /// </remarks>
+#if !ES_BUILD_STANDALONE
+    [DynamicallyAccessedMembers(ManifestMemberTypes)]
```

Comment:
It maybe nice to add a comment on this line saying that it is used by `EnsureDescriptorsInitialized()`.